src/components/Footer.js

```diff
@@ -6,28 +6,40 @@ class Footer extends Component {
 		return (
 			<footer className="footer">
 				<span className="todo-count">
-					<strong>99</strong> items left
+					<strong>{this.props.remaining}</strong> items left
```

Comment:
You could add in a bit of extra logic to have the wording dynamic.  For example "50 items left" (plural) and "1 item left" (singular).